README.md

```diff
@@ -80,7 +80,11 @@ If you wish, you can tweak settings by editing the release yaml.
 This file can then be installed into your management cluster with `kubectl apply -f _out/control-plane-components.yaml`.
 
 Note that CACPPT should be deployed as part of a set of controllers for Cluster API.
-You will need at least the upstream CAPI components, the Talos bootstrap provider, and an infrastructure provider for v1alpha3 CAPI capabilities.
+You will need at least the upstream CAPI components, the Talos bootstrap provider, and an infrastructure provider for v1beta1 CAPI capabilities.
+
+CACPPT plays the following role in the whole Cluster API architecture:
+
+![Cluster API CACPPT](/docs/images/cacppt.jpg)
```

Comment:
notes on the diagram:

* `.jpg` -> `.png` (add a source file?)
*  `Machines Controllers` -> should it be `MachineController` ?
* colors for core CAPI/CABPT?
* `Bootstrap` action should go to a single node only (arrows right now are confusing)
* CABPT generates bootstrap data for any `Machine`
* `Talos Control Plane` is a bit confusing? node? `TalosControlPlane` resource? if these are nodes, we could show them as nodes, and they have matching CAPI resources?
* `Initialize control planes` ? there's just a single control plane for the cluster which consists of several nodes
* `Infrastructure Controllers` -> `Infrastructure Controller` (there's a single one?); infrastructure controller passes bootstrap data from CABPT to the nodes
* arrows in general -> `MachineController` -> `InfrastructureController`: what does this arrow represent? These two controllers don't communicate directly. There are resources in between which are not in the picture.